rskj-core/src/main/java/co/rsk/net/handler/TxsPerAccount.java

```diff
@@ -32,8 +32,10 @@
 
     boolean containsNonce(BigInteger nonce) {
         for (Transaction tx : txs) {
-            if (new BigInteger(1, tx.getNonce()).equals(nonce)) {
-                return true;
+            for(byte[] txNonce : tx.getNonces()) {
```

Comment:
I suspect this is not what is intended. For a specific account there is a single relevant nonce.  We should look who is using TxsPerAccount and find how having multiple nonces for a txs affect that.